Making the winning requirements for Beta end more clear

highlander1599 opened the pull request:
Some discussions on Discord showed that the winning requirements for Beta end are not precise enough for players that are used to not playing the level until the timer ends. Therefore the Extra Objective Message includes now the requirement to survive until the timer ends.

kreuvf commented on po/de.po:
```diff
@@ -6347,7 +6347,7 @@ msgstr "Gasturbinengenerator Mk2"
 
 #: po/custom/fromJson.txt:506
 msgid "Gas Turbine Generator Mk3"
-msgstr "Gasturbinengenerator Mk3"
```
"Mk3" ist doch total out.

highlander1599 committed:
Making the winning requirements for Beta end more clear

Some discussions on Discord showed that the winning requirements for Beta end are not precise enough for players that are used to not playing the level until the timer ends. Therefore the Extra Objective Message includes now the requirement to survive until the timer ends.
